content/posts/09-nibbler-flatpak.md

```diff
@@ -0,0 +1,230 @@
+---
+title: "Flatpak-ing Nibbler, a Leela Chess Zero Interface and some Neural Network Engines"
+date: 2020-10-30T17:39:23-03:00
+draft: false
+---
+
+> Article by editor Leo <thinkabit.ukim@gmail.com>  
+
+> The Machine Learning terminology here is simplified and will have a loss
+> quality, but not pertinent to the point.
+
+Nibbler is a Graphical User Interface for analysis of chess games using mainly
+the [Leela Chess Zero][1] chess engine (computer that plays chess and provides
+an evaluation of positions), it is written specifically for using Leela.
+
+It deals with Leela's output which is incompatible with traditional chess engines
+and the Chess GUIs that support the latter, that specific support allow it to
```

Comment:
Leela's output is incompatible with Chess GUIs that support the output of traditional chess engines

Reply:
> It deals with Leela's output which is incompatible with traditional chess engines
and the Chess GUIs that support the latter

Sorry? Leela's output is different from what traditional chess engines output, that I get, but I'm not quite sure what you're trying to say with:

> and the Chess GUIs that support the latter

Reply:
Maybe you could word this differently then? It's a bit confusing otherwise